src/snapred/backend/data/GroceryService.py

```diff
@@ -339,8 +444,8 @@ def fetchNeutronDataCached(self, runId: str, useLiteMode: bool, loader: str = ""
         self._loadedRuns[key] += 1
         return data
 
-    def fetchLiteDataMap(self) -> WorkspaceName:
-        item = GroceryListItem.builder().grouping("Lite").build()
+    def fetchLiteDataMap(self, runId: str) -> WorkspaceName:
```

Comment:
The Lite data map really shouldn't be dependent on run ID, nor on state.  Is there a way to remove this dependency?

Reply:
The dependency has now been removed, by using some reserved locations in the instrument-donor cache system (and adding a "special case" to the loading of the lite data map).  Note that, as I mentioned on slack, the lite-data map workspace won't have the correct location parameters -- right now, I am assuming that this doesn't matter for its intended usage.  (When it does matter, it's still possible to specify a run number for it, and the corresponding location parameters will be updated in a cached copy.)